### ISSUE TITLE
Uncaught TypeError: Cannot assign to read only property 'include' of latlong

### ISSUE THREAD

**Issue by AlexCppns** [marioizquierdo/jquery.serializeJSON#29]
line 186 in the js file:

      o[key] = [];

I am not sure what does that.

**Comment by dedepress** [comment]
same problem, try code below to get error :)

sometimes, 

<pre>

 <input type="hidden" name="address" value=""/>

  <input type="text" name="address[city]"         value="San Francisco" />
  <input type="text" name="address[state][name]"  value="California" />
  <input type="text" name="address[state][abbr]"  value="CA" />
</pre>

**Comment by marioizquierdo** [comment]
This one makes sense. The first (hidden) field is assigning "address" as an empty string. Then the next fields will try to add keys to that string assuming it should be an object.

If you do not have control over what invalid field has been added to the form, I would recommend to remove that one from the jQuery set in the JavaScript side. Maybe something like this:

```javascript
jQuery('form input').not('input[type=hidden][name=address]').serializeJSON();
```

Or, use your own parser function, like this:

```javascript
var handleAddedHiddenField = function(val, inputName) {
  return inputName == "address" && val === "" ? return {} : val;
}

jQuery('form').serializeJSON(parseWithFunction: handleAddedHiddenField);
```

**Comment by marioizquierdo** [comment]
This was fixed in the last release 2.5.0.

Note that the last value will override the previous values, but at least it will not crash anymore.
